python-backend/app/api/mines/mine_api_models.py

```diff
@@ -173,6 +173,7 @@ def format(self, value):
 MINE_INCIDENT_RECOMMENDATION_MODEL = api.model(
     'Mine Incident Recommendation', {
         'recommendation': fields.String,
+        'mine_incident_recommendation_guid': fields.String
```

Comment:
if we're passing in a dictionary, why isn't 'Mine Incident Recommendation' put in the dictionary?  Is it a particular patern?